terraform/modules/cluster/addons.tf

```diff
@@ -236,6 +237,39 @@ module "eks_blueprints_kubernetes_addons" {
     additional_irsa_policies = ["arn:aws:iam::aws:policy/AdministratorAccess"]
   }
 
+  argocd_helm_config = {
```

Comment:
Can we make sure the Argo components run on the system node group? See this example where we do it for the load balancer controller https://github.com/aws-samples/eks-workshop-v2/blob/main/terraform/modules/cluster/addons.tf#L102

Reply:
Used approach from PR 354